### PR TITLE
Feature/recipes

### PR DESCRIPTION
**Pull request by pdemaio-travelperk**

### CODE REVIEW

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (israel-tk) [inline .idea/inspectionProfiles/profiles_settings.xml]: avoid adding `.idea` files to the commited files in the repo. You already added this folder to the `.gitignore` file, but I guess it was after adding these files

File: app/recipe/tests/test_recipe_api.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+def sample_ingredient(recipe: Recipe, name: str = "Cucumber") -> Ingredient:
+    return Ingredient.objects.create(name=name, recipe=recipe)
+
+
+def url_for_recipe(recipe_id: str):
+    return reverse('recipe:recipe-detail', args=[recipe_id])
+
+
+class RecipeTests(TestCase):
```
Comment (israel-tk) [inline app/recipe/tests/test_recipe_api.py:27]: You might want to test some edge cases:
- Getting a non-existent recipe by id (should return 404).
- Create a recipe without name, description or ingredients (should this be valid?)
- Create or update the ingredients of a recipe with an empty ingredient name.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Django>=4.0.0,<4.1.0
```
Comment (israel-tk) [inline requirements.txt]: Normally in Python in `requirements.txt` file you pin the packages to an exact version, rather than a range of versions. This ensures repeatable builds among different environments.

Read https://nvie.com/posts/pin-your-packages/ to learn why.

**Review by israel-tk — APPROVED** [review]
Great work! Just left some comments as suggestions and ideas.

File: requirements.txt (hunk 3 above)
Comment (pdemaio-travelperk) [reply requirements.txt]: done!

File: .idea/inspectionProfiles/profiles_settings.xml (hunk 1 above)
Comment (pdemaio-travelperk) [reply .idea/inspectionProfiles/profiles_settings.xml]: good catch, thought github's .gitignore was already taking care of that

File: app/recipe/tests/test_recipe_api.py (hunk 2 above)
Comment (pdemaio-travelperk) [reply app/recipe/tests/test_recipe_api.py:27]: done!
